sql_metadb/report_queries/erm/count/erm_agreement_count.sql

```diff
@@ -25,12 +24,12 @@ WITH parameters AS (
         NULL::DATE AS ent_start_date,
         NULL::DATE AS ent_end_date,
         --'2021-01-01' :: DATE AS ent_start_date, -- start date day is included in interval
-        --'2022-01-01' :: DATE AS ent_end_date, -- end date day is NOT included in interval -> enter next day	
+        --'2022-01-01' :: DATE AS ent_end_date, -- end date day is NOT included in interval -> enter next day
 		-- filters on package content item level
         NULL::DATE AS pci_start_date,
         NULL::DATE AS pci_end_date
         --'2021-01-01' :: DATE AS pci_start_date, -- start date day is included in interval
-        --'2022-01-01' :: DATE AS pci_end_date, -- end date day is NOT included in interval -> enter next day		
+        --'2022-01-01' :: DATE AS pci_end_date, -- end date day is NOT included in interval -> enter next day
 )
 
```

Comment:
Please delete empty line.

Reply:
This is a very resisting line ... ;)
Removed it now.